velox/duckdb/conversion/tests/DuckWrapperTest.cpp

```diff
@@ -188,14 +211,98 @@ TEST_F(BaseDuckWrapperTest, tpchSF1) {
   // test TPC-H loading and querying SF0.01
   execute("CALL dbgen(sf=0.01)");
   // test conversion of date, decimal and string
-  verifyUnaryResult<double>("SELECT l_discount FROM lineitem LIMIT 1", {0.04});
+  verifyUnaryResult<ShortDecimal>(
+      "SELECT l_discount FROM lineitem LIMIT 1", {ShortDecimal(4)});
   verifyUnaryResult<Date>(
       "SELECT l_shipdate FROM lineitem LIMIT 1", {Date(9568)});
   verifyUnaryResult<StringView>(
       "SELECT l_comment FROM lineitem LIMIT 1",
       {StringView("egular courts above the")});
 }
 
+TEST_F(BaseDuckWrapperTest, duckToVeloxDecimal) {
+  // test SMALLINT conversion.
```

Comment:
nit: Comments should be full sentences, i.e. start with a capital letter and end with a period. Please, update throughout.